stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
+  gutter-position: inside-static,
+  column-width: 96px,
+  gutters: $gutter-width/96px,
+  // debug: (
+  //   image: show-columns,
+  //   output: overlay,
+  //   toggle: bottom left,
+  // )
+);
+
+%l-container,
+.l-container {
+  //@include container($breakpoint-largest);
+  @include clearfix;
+  max-width: $breakpoint-large;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: $body-gutter - gutter();
+  padding-right: $body-gutter - gutter();
+  .no-mediaqueries & {
+    min-width: $breakpoint-no-mediaqueries;
+  }
+  //Scale up
+  @media (min-width: $breakpoint-large + 1) {
```

Comment:
I think changing the bleed at specific breakpoints is possibly too design specific for a boilerplate. Thoughts?